compute_endpoint/globus_compute_endpoint/endpoint/endpoint_manager.py

```diff
@@ -453,7 +460,12 @@ def cmd_start_endpoint(
             startup_proc_title = f"Endpoint starting up for {uname} [{args_title}]"
             setproctitle.setproctitle(startup_proc_title)
 
-            amqp_creds = json.dumps(kwargs.get("amqp_creds"))
+            stdin_data_dict = {
+                "amqp_creds": kwargs.get("amqp_creds"),
+                "config": render_config_user_template(self.conf_dir, user_opts),
```

Comment:
A point of reference that we'll want to take care to explicitly document: this detail happens _after_ the child process has dropped privileges.

Not for this PR, I don't think, but something to keep in our minds.